src/sphinx_book_theme/assets/scripts/index.js

```diff
@@ -55,22 +49,50 @@ var toggleFullScreen = () => {
   }
 };
 
-// Enable tooltips
-var initTooltips = () => {
-  $(document).ready(function () {
-    $('[data-toggle="tooltip"]').tooltip();
-  });
+/**
+ * Sidebar scroll on load.
+ *
+ * Detect the active page in the sidebar, and scroll so that it is centered on
+ * the screen.
+ */
+var scrollToActive = () => {
+  var navbar = document.getElementById("site-navigation");
+  var active_pages = navbar.querySelectorAll(".active");
+  var active_page = active_pages[active_pages.length - 1];
+  // Only scroll the navbar if the active link is lower than 50% of the page
+  if (
+    active_page !== undefined &&
+    active_page.offsetTop > $(window).height() * 0.5
+  ) {
+    navbar.scrollTop = active_page.offsetTop - $(window).height() * 0.2;
+  }
 };
 
 /**
- * Hide the Table of Contents any time sidebar content is on the screen.
+ * Functionality that is triggered before / after printing the page.
```

Comment:
The function itself is just called on button click, so this comment might not be exactly well placed? The text inside the function dwells more into the before / after part of the print function itself.

Reply:
gotcha - will clarify this 👍